src/modules/item/Item.tsx

```diff
@@ -188,27 +181,24 @@ const DocumentContent = ({
   return documentItem;
 };
 
-const AppContent = ({ item }: { item: AppItemTypeRecord }): JSX.Element => {
-  const {
-    data: member,
-    isLoading: isLoadingMember,
-    isSuccess: isSuccessMember,
-  } = useCurrentMemberContext();
+const AppContent = ({ item }: { item: AppItemType }): JSX.Element => {
+  const { data: member, isLoading: isLoadingMember } =
+    useCurrentMemberContext();
   const { t: translateMessage } = useMessagesTranslation();
 
   if (isLoadingMember) {
     return (
       <Skeleton variant="rectangular" width="100%" height={SCREEN_MAX_HEIGHT} />
     );
   }
-  if (isSuccessMember)
+  if (member)
```

Comment:
Ideally we should switch the order of these 2 conditions to avoid loading when re-fetching. First checking for data and then for loading state.